pkg/virtctl/vmexport/vmexport_test.go

```diff
@@ -89,6 +77,30 @@ var _ = Describe("vmexport", func() {
 			portChan <- uint16(5432)
 			return nil
 		}
+
+		secret = &v1.Secret{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: secretName,
+			},
+			Type: v1.SecretTypeOpaque,
+			Data: map[string][]byte{
+				"token": []byte("test"),
+			},
+		}
+
+		vme = &exportv1.VirtualMachineExport{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: vmeName,
```

Comment:
nit: VME and Secret should be in the same namespace. Maybe set `Namespace` to make it explicity?

Reply:
In the end it is up to the client, so I don't think it'll make a difference setting it here.